gateway-service/src/main/java/org/zowe/apiml/gateway/controllers/GatewayHomepageController.java

```diff
@@ -37,28 +37,30 @@ public class GatewayHomepageController {
     private static final String SUCCESS_ICON_NAME = "success";
     private static final String WARNING_ICON_NAME = "warning";
     private static final String UI_V1_ROUTE = "%s.ui-v1.%s";
-    private static final String ZAAS_SERVICEID = CoreService.ZAAS.getServiceId();
 
     private final DiscoveryClient discoveryClient;
-
     private final BuildInfo buildInfo;
-    private String buildString;
 
     private final String apiCatalogServiceId;
+    private String buildString;
+
+    private final String serviceId;
 
     @Autowired
     public GatewayHomepageController(DiscoveryClient discoveryClient,
```

Comment:
Why are there two constructors? There could be just lombok annotation, annotated fields and call of `initializeBuildInfos` in @PostCostruct.